packages/cactus-plugin-bungee/src/test/typescript/fabric-contracts/simple-asset/chaincode-typescript/package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "asset-transfer-basic",
+    "version": "1.0.0",
+    "description": "Asset Transfer Basic contract implemented in TypeScript",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "engines": {
+        "node": ">=12",
+        "npm": ">=5"
+    },
+    "scripts": {
+        "lint": "tslint -c tslint.json 'src/**/*.ts'",
+        "pretest": "npm run lint",
+        "test": "nyc mocha -r ts-node/register src/**/*.spec.ts",
+        "start": "fabric-chaincode-node start",
+        "build": "tsc",
+        "build:watch": "tsc -w",
+        "prepublishOnly": "npm run build"
+    },
+    "engineStrict": true,
+    "author": "Hyperledger",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "fabric-contract-api": "2.2.2",
+        "fabric-shim": "2.2.2"
+    },
+    "devDependencies": {
+        "@types/chai": "4.3.0",
+        "@types/mocha": "5.2.7",
+        "@types/node": "18.11.9",
+        "@types/sinon": "5.0.7",
+        "@types/sinon-chai": "3.2.8",
+        "chai": "4.3.6",
+        "mocha": "5.2.0",
+        "nyc": "14.1.1",
+        "sinon": "7.5.0",
+        "sinon-chai": "3.7.0",
+        "ts-node": "7.0.1",
+        "tslint": "5.20.1",
+        "typescript": "5.3.3"
```

Comment:
@eduv09 Please remove the ones from here that aren't strictly necessary for the test case to remain operational (such as chai, mocha, nyc, sinin, ts-node, tslint and most likely typescript as well). The point being that these dependencies will have to be maintained and upgraded because our security automation tooling will catch vulnerabilities in them in the future and then it adds the burden on everybody else later to fix those so it's better to strip down these exmaples that are just for automated tests as much as possible.

Reply:
@petermetz is there a way to automatically identify unused packages? [i found depcheck](https://stackoverflow.com/questions/22675725/how-to-find-unused-packages-in-package-json), but does not seem canonical to the project. We could add a dependency, add an npm task to check unused deps, and update the PR for contributors to run depcheck before opening a PR.

Reply:
@RafaelAPB I agree on all of those. Some more context is needed though: We tried depcheck many years ago and it would identify packages to remove that were not actually removable, e.g. we acted on it's advice and the build and/or tests broke.

With all that said, this was close to 5 years ago so the tool could've improved in the meantime and our build process also definitely improved in the meantime (we went through several major upgrades on npm and then yarn as the package manager).

Therefore trying it again seems like a rational thing to do because it could easily be that nowadays "it just works". 

Also especially agree that if we do find a tool (depcheck or any other) that correctly identifies extraneous dependencies then that should be part of the CI so that it's not possible to accidentally commit such dependencies to main anymore.